tests/test_account_chart.py

```diff
@@ -185,3 +205,50 @@ def test_update_account_with_invalid_group_raise_error():
         cashctrl_ledger.update_account(account=1149, currency='USD',
             text='test create account', vat_code='MwSt. 2.6%', group='/ABC'
         )
+
+# Tests the mirroring functionality of accounts.
+def test_mirror_accounts(add_and_delete_vat_code):
+    target_df = (pd.read_csv('tests/initial_accounts.csv', skipinitialspace=True))
```

Comment:
The CashCtrl account chart is pretty hard to read in csv format because it is defined in multiple languages.

How can I know it is different from the default chart on the server?
(Otherwise, the test would only check whether the data remains unchanged)

As an alternative, we could use `TestLedger.account_chart()` defined on [pyledger/test_ledger.py#L17-L80](
https://github.com/macxred/pyledger/blob/7d28cbea70c5b99b527fe6db296473563beffa75/pyledger/test_ledger.py#L17-L80). We would need to add a 'group' column to this account chart, tough.

Let's discuss this proposal in tomorrow's call.